conf.d/haproxy.yaml.example

```diff
@@ -5,6 +5,10 @@ instances:
     # username: username
     # password: password
     #
+    # or, with a unix stats or admin socket:
+    #
+    # url: unix:///var/run/haproxy.sock
```

Comment:
Nitpick: could you add a leading `-` to the example so it's clear that should be another instance and not an attribute of the first one?
```
 instances:
   - url: http://localhost/admin?stats
      # username: username
      # password: password
   # or, with a unix stats or admin socket:
   # - url: unix:///var/run/haproxy.sock
```